Kudu.Services/Functions/FunctionController.cs

```diff
@@ -22,6 +25,100 @@ public FunctionController(IFunctionManager manager, ITraceFactory traceFactory)
             _traceFactory = traceFactory;
         }
 
+        [HttpPut]
```

Comment:
I never know with WebAPI. Sometimes I need to have it even though I have the HttpMethodConstraints, but I'm not sure. I see it in all of kudu's controllers though.